examples/src/main/java/io/grpc/examples/advanced/HelloJsonClient.java

```diff
@@ -43,7 +43,7 @@
  * https://groups.google.com/forum/#!forum/grpc-io
  */
 public final class HelloJsonClient {
-  private static final Logger logger = Logger.getLogger(HelloWorldClient.class.getName());
```

Comment:
it looks like the import for this class is no longer needed. (same for the other class)

Reply:
oh，maybe you can just remove it.

Reply:
yeah, you can remove line 30 because the import of HelloWorldClient is no longer used in this class.